source/_integrations/plex.markdown

```diff
@@ -97,6 +98,39 @@ The library sensors are disabled by default, but can be enabled via the Plex int
   
 </div>
 
+## Button
+
+A `button.scan_clients` entity is available to discover new controllable Plex clients. This may be necessary in scripts or automations which control a Plex client app, but where the underlying device must be turned on first. This button is preferred over the legacy `plex.scan_for_clients` service.
+
+Example script:
+
+{% raw %}
+
+```yaml
+play_plex_on_tv:
+  sequence:
+    - service: media_player.select_source
+      target:
+        entity_id: media_player.smart_tv
+      data:
+        source: "Plex"
+    - wait_template: "{{ is_state('media_player.smart_tv', 'On') }}"
```

Comment:
```suggestion
    - wait_template: "{{ is_state('media_player.smart_tv', 'on') }}"
```

Reply:
Actually, this should be replaced with a wait-for trigger without templating.

Reply:
Hmm, I just updated this but now I'm reconsidering. A `wait_for_trigger` will not fire if the state was already correct and the timeout will impose an unnecessary delay.

Reply:
Right... would be possible to wrap it with an `if` statement, however, that is not available in the current release (added in 2022.5).